modules/social_features/social_event/modules/social_event_an_enroll/social_event_an_enroll.install

```diff
@@ -89,8 +89,8 @@ function _social_event_an_enroll_fix_blocks() {
  */
 function social_event_an_enroll_update_8001() {
   $config = \Drupal::service('config.factory')->getEditable('social_event_an_enroll.settings');
-  $config->set('event_an_enroll_email_subject', "You are enrolled in the event [event name]");
```

Comment:
@ribel If I'm correct the `[event name]` tokens did not work right?
I guess now we will now have to supply an update hook which updates these tokens to the new version. Not just setting a new default but actually updating the active value.

Any reason why these tokens were changed to `[node:title]` and `[social_event_an_enroll:enrolled_event]`? They do not seem very understandable for non-technical site managers.

Reply:
Clear!

Imho I think adding a custom token will be much more useful for site managers.

Reply:
I think that's outside of the scope of this issue though. Could you create a follow-up? :)

Reply:
Agreed, but we are changing it in this feature right?

Reply:
No the different tokens are already committed but the documentation string wasn't updated. That's what's being fixed here. So basically this feature also includes a bugfix. Simplifying the tokens further would be a new feature :)